site/content/docs/5.1/examples/navbar-sticky/navbar-sticky.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+  'use strict'
+
+  function switchHeader() {
+    var element = document.getElementsByTagName('header')[0]
```

Comment:
I don't really know why, but `let` appears as an error here because it's a reserved keyword. I tried to copy Bootstrap on this file and it looks like they use `var` everywhere in those js examples files